doc/source/user/plugins.rst

```diff
@@ -254,6 +254,32 @@ files. For example::
 
     results_reader = serpentTools.ResultsReader(str(result.outputs[-1]))
 
+
+ABCE Plugin
+++++++++++++
+
+The :class:`~watts.PluginABCE` class enables simulations with the Agent Based Capacity
+Expansion (`ABCE`) code using a templated input file. Since :mod:`watts` relies on the `Jinja
+<https://jinja.palletsprojects.com>`_ templating engine, any parameter in the `ABCE` settings
```

Comment:
```suggestion
Expansion (ABCE) code using a templated input file. Since :mod:`watts` relies on the `Jinja
<https://jinja.palletsprojects.com>`_ templating engine, any parameter in the ABCE settings
```